### PR TITLE
Releaseing V2.1.2 of metadata with updated data release HTML file

### PR DESCRIPTION
**Pull request by nsamanta**
Added the updated metadata spreadsheet to `public/assets/metadata` and the updated `index.html` file in `public/history`

### CODE REVIEW

File: history/index.html [hunk 1]
```diff
@@ -644,4 +757,4 @@ <h3 style="color: #993366;">Data Release</h3>
   </ul>
 </body>
 
-</html>
+</html>
```
Comment (yimsheng) [inline history/index.html:760]: empty line at the end of the file

File: history/index.html [hunk 2]
```diff
@@ -3,636 +3,749 @@
...
   <title>Data Release</title>
 </head>
 
 <body>
   <h3 style="color: #993366;">Data Release</h3>
   <ul>
+    <li><a
+        href="https://hpap.pmacs.upenn.edu/assets/metadata/PancDB_metadata_V2.1.2.zip">
+        V2.1.2
+      </a>
```
Comment (yimsheng) [inline history/index.html:16]: date

**Review by yimsheng — COMMENTED** [review]
looks great

**Comment by nsamanta** [comment]
@yimsheng I just made a new commit with minor changes. Can you let me know if everything looks good?

**Comment by yimsheng** [comment]
@nsamanta 
Newline at the end of the file is needed. You can enable this setup in vscode. 

You can take a look at this note for setup:https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3

By the way, what does the new file history/index.html:Zone.Identifier do?

**Comment by nsamanta** [comment]
> @nsamanta Newline at the end of the file is needed and you can see the red circle with minus sign in it at the end of the file if newline is missing. You can enable this setup in vscode.
> 
> You can take a look at this note for setup:https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3
> 
> By the way, what does the new file history/index.html:Zone.Identifier do?

@yimsheng Okay I will add the newline in VS code with the setting. I have no idea about the `zone identifier` files, they keep popping up as `unstaged` files.

**Comment by nsamanta** [comment]
@yimsheng I deleted the zone identifier file and made a new commit

**Comment by dongbohu** [comment]
Look at the lines 20, 44, 84, etc of `index.html`, can't you see the problem?

File: history/index.html [hunk 3]
```diff
@@ -3,636 +3,750 @@
...
-    content="text/html; charset=utf-8"
-  />
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
   <title>Data Release</title>
 </head>
 
 <body>
   <h3 style="color: #993366;">Data Release</h3>
   <ul>
+    <li><a
```
Comment (yimsheng) [inline history/index.html:13]: This li should be outside the ul like the rest of the items

Comment (yimsheng) [inline history/index.html:13]: this li should be outside the ul and the closing li tag is missing

Comment (yimsheng) [reply history/index.html:13]: please make sure all html tag has the corresponding closing tag. you can check this by folding the tags in vscode if the tags are matching.
If there's any unmatching tags, the formatter cannot compile

File: history/index.html [hunk 4]
```diff
@@ -3,636 +3,750 @@
...
-        <a href="https://hpap.pmacs.upenn.edu/assets/metadata/PancDB_archived_metadata.zip">
-          PANC-DB archived metadata
+        <a href="https://faryabi16.pmacs.upenn.edu/hpap-history/V2.0.1">
+          removed files from V2.0.1
         </a>
       </li>
+    </ul>
+    <br />
+
+  <li>
```
Comment (yimsheng) [inline history/index.html:382]: missing closing tag

**Comment by nsamanta** [comment]
Okay I identified several extra tags etc, I am working on it now. Will make a new commit with these corrections

**Comment by nsamanta** [comment]
@dongbo the latest commit by @yimsheng should have the most up-to-date index.html file

**Pull request merged**